go/fn/examples/example_asmain_runner_test.go

```diff
@@ -48,12 +50,14 @@ func (r *SetLabels) Run(ctx *fn.Context, functionConfig *fn.KubeObject, items fn
 //   - apiVersion: v1
 //     kind: Service
 //     metadata:
-//       name: example
+//     name: example
```

Comment:
nit: seems like name should be indented?

Reply:
Thanks! fix.